app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'application_record'
+
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you use `after_save :update_comments_counter` 👍